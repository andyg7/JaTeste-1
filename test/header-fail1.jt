#include_jtlib "nvlkj"
func int main()
{
	return 0;
}
