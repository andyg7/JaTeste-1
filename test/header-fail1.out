Scanned
Parsed
Fatal error: exception Exceptions.InvalidHeaderFile("./nvlkj")
